app/views/clinical_resource_hubs/_crh_categories_combo_box.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="usa-combo-box grid-col-4 practice-<%= practice_type %>-categories-combo-box" data-placeholder="All categories">
+  <label for="crh-search-<%= practice_type %>-categories" class="line-height-26">Categories</label>
+  <select id="crh-search-<%= practice_type %>-categories" class="usa-select" name="crh-search-<%= practice_type %>-categories">
+    <% if practice_categories.count > 0 %>
```

Comment:
can we wrap the if statement around the entire div and add `practice_categories.present? && count check`

Reply:
done.